entries/u/uber.com.json

```diff
@@ -3,13 +3,12 @@
     "domain": "uber.com",
     "url": "https://www.uber.com/",
```

Comment:
```suggestion
```

The `url` key is only needed if going to the domain (e.g. `https://uber.com`) fails